README.md

```diff
@@ -133,7 +134,9 @@ On the master:
 * This is the string contents for the authorized keys for the slaves to push into the repo.
   It should match the `jenkins::private_ssh_key` on the master.
   * `jenkins::private_ssh_key`
-  * The key which authorizes access to push content into the repository or to connect back to the master from a job.
+  * The ssh private key which authorizes access to push content into the repository or to connect back to the master from a job. 
```

Comment:
I am not sure anymore but is it currently being used to "connect back to the master from a job"?